docs/src/proposals/optimistic-transaction-propagation-signal.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Optimistic Transaction Propagation Signal
```

Comment:
I think the document could expand with more detail on how (after retransmit stage) we will infer "Optimistic Execution" for shreds, and what would happen then.

Reply:
The intention is to expose some new commitment level like confirmed/processed to indicate some level of distribution across staked nodes.